index.rst

```diff
@@ -44,14 +44,628 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
+
+.. _charge: https://sitcomtn-013.lsst.io/
+.. _DMTN-126: https://DMTN-126.lsst.io/
+.. _RSP: https://nb.lsst.io/
+
 .. note::
 
    **This technote is not yet published.**
 
-   The First-Look Analysis and Feedback Functionality Breakout Group was `charged <https://sitcomtn-013.lsst.io/>`_ with identifying how we'll perform on-the-fly analysis and display of both local data reduction artifacts and camera images. This document details the answers to the charge questions and other findings.
+   The First-Look Analysis and Feedback Functionality Breakout Group `charge`_ details questions regarding how the Rubin Project will perform on-the-fly analysis and display of both local data reduction artifacts and camera images. This document details the answers to the charge questions and other findings.
+
+
+Executive Summary
+=================
+
+This is the initial report by the FAFF working group and contains all of the findings and recommendations that will be provided in the final copy.
+Due to the current summit power issues, aspects of the document could not be completed at this time, specifically the documentation of the proof-of-concept demonstrators due to the loss of the Andes cluster and current void of data at the summit.
+
+The working group accumulated a series of use-cases that represent the functionality required to successfully perform on-sky and on-the-fly operations during the commissioning phase.
+By analyzing the use-cases and exploring how they would be accomplished using our current toolset, it has been determined that a large fraction of the necessary capabilities are already scoped and implementend but require augmenting certain functionalities to enable proper interfacing and inter-operability.
+However, there exist a important gaps that require new software, selection of adopted packages, and the development of examples such that users can use the components effectively. 
+This is especially apparent with regards to on-the-fly analysis tasks and ability to support the diversity of figures that will be required for monitoring, system characterization and even global system debugging.
+A new CSC will need to be developed to handle the creation and management of analysis tasks as well as the alerting of newly created artifacts to users.
+A design and scope estimate for this CSC has been developed and all the information is ready to support the filing of an LCR if required.
+Dynamic figures that are to be embedded into the LOVE system will be required to use the Bokeh package to create an application, which will then be deployed using Kubernetes.
```

Comment:
Suggest "We recommend that dynamic figures that are to be embedded in the LOVE system use the Bokeh package to create an application, which will then be deployed using Kubernetes."

Reply:
I wonder if this phrase should be moved to just after "This is especially apparent with regards..." above in line 68. That phrase is talking about visualization/figures, then it goes to talk about new CSC and then back to vis/fig. 

Suggest over suggest:

> We demonstrated that dynamic and interactive visualizations can be created using the Bokeh (add link) framework, that can embedded into LOVE and deployed using Kubernetes.